v21.2/stream-data-out-of-cockroachdb-using-changefeeds.md

```diff
@@ -810,6 +810,119 @@ In this example, you'll set up a changefeed for a single-node cluster that is co
     $ cockroach quit --insecure
     ~~~
 
+### Create a changefeed connected to a webhook sink
+
+{{site.data.alerts.callout_info}}
+[`CREATE CHANGEFEED`](create-changefeed.html) is an [enterprise-only](enterprise-licensing.html) feature. For the core version, see [the `CHANGEFEED FOR` example above](#create-a-core-changefeed).
+{{site.data.alerts.end}}
+
+{% include {{ page.version.version }}/cdc/webhook-beta.md %}
+
+<span class="version-tag">New in v21.2:</span> In this example, you'll set up a changefeed for a single-node cluster that is connected to a local HTTP server via a webhook. For this example, you'll use an [example HTTP server](https://github.com/cockroachlabs/cdc-webhook-sink-test-server/tree/master/go-https-server) to test out the webhook sink.
+
+1. If you do not already have one, [request a trial enterprise license](enterprise-licensing.html).
```

Comment:
Dumb questions: 

- do Cloud customers already have an enterprise version?  Is it worth mentioning this here?
- do Cloud customers even know what an enterprise license is?
- Does this functionality work with a Cloud instance?

Reason I'm asking is because it might be confusing if a Cloud customer tries to do this and then goes "wait what is enterprise, and do I have it?"  Maybe this could be mitigated by adding something like "Cloud users will already have a license" (with Cloud linking to https://www.cockroachlabs.com/docs/cockroachcloud/ ?